### PR TITLE
Add GitHub Actions 

### PR DESCRIPTION
**Pull request by danfowler**
- [x] Set up automated testing for this repository using GitHub Actions.
- [x] Add test scheme to repo
- [x] Fix broken tests

### CODE REVIEW

File: OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift [hunk 1]
```diff
@@ -4,7 +4,7 @@ import OmiseSDK
 
 @available(*, deprecated)
 class OmiseTokenRequestTest: XCTestCase {
-    private let publicKey = "<#Omise Public Key#>"
+    private let publicKey = "pkey_test_58wfnlwoxz1tbkdd993"
```
Comment (danfowler) [inline OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift:7]: Should this somehow be stored in an environment variable? 🤔

Comment (tobias90) [reply OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift:7]: Yes I think that would be nice. Now we have this key in three different files. 
Is it possible to securely store keys in GitHub Actions so we can remove it from the repository?

Comment (danfowler) [reply OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift:7]: Yes 

https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Comment (tobias90) [reply OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift:7]: Okay nice. But I think there is not straightforward way to set the environment variables from xcodebuild. Let's look into this later and create another PR.

Comment (igroomgrim) [reply OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift:7]: Seems to look similar to the environment variable in Node.js 😄

**Pull request merged**